CedarJava/src/main/java/com/cedarpolicy/model/ValidationResponse.java

```diff
@@ -28,62 +28,87 @@
 
 /** Result of a validation request. */
 public final class ValidationResponse {
-    /** Exactly one of `results` or `errors` should be None. */
+    /** Is this a success or a failure response */
+    @JsonProperty("type")
+    public final SuccessOrFailure type;
+    /** This will be present if and only if `type` is `Success`. */
     public final Optional<ValidationResults> results;
```

Comment:
Nit: To match the naming in AuthorizationReponse.java
```suggestion
    public final Optional<ValidationSuccessResponse> success;
```

Reply:
Renamed and pushed